pkg/scheduler/framework/plugins/interpodaffinity/scoring.go

```diff
@@ -224,13 +224,13 @@ func getPreScoreState(cycleState *framework.CycleState) (*preScoreState, error)
 func (pl *InterPodAffinity) Score(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod, nodeName string) (int64, *framework.Status) {
 	nodeInfo, err := pl.sharedLister.NodeInfos().Get(nodeName)
 	if err != nil || nodeInfo.Node() == nil {
-		return 0, framework.NewStatus(framework.Error, fmt.Sprintf("getting node %q from Snapshot: %v, node is nil: %v", nodeName, err, nodeInfo.Node() == nil))
+		return 0, framework.AsStatus(fmt.Errorf("failed to get node %q from Snapshot: %w, node is nil: %v", nodeName, err, nodeInfo.Node() == nil))
```

Comment:
The `nodeName` here cannot be removed since we run score plugins in parallelism and aggregate error infos. But just find out that such log https://github.com/kubernetes/kubernetes/blob/e3fbbd33d6616a072833ecb2489b48d4f30ec3d4/pkg/scheduler/framework/runtime/framework.go#L752
 may be duplicated since they will be logged again in scheduler.go again like https://github.com/kubernetes/kubernetes/blob/7315c1f6ddf37e41ae7826a042d057d5f594c10c/pkg/scheduler/scheduler.go#L483.

Reply:
I think we have removed any cases were `nodeInfo.Node() == nil` in the plugins (Nodes in the snapshot that return nil here are not included in lists). Do a quick check and leave as:

```
return 0, framework.AsStatus(fmt.Errorf("failed to get node %q from Snapshot: %w", nodeName, err))
```

Reply:
confirmed and updated.